lib/widget.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/foundation.dart';
 import './story.dart';
-import './property_widgets.dart';
+import './property_widgets.dart' as p;
```

Comment:
Any reason why you needed to make that alias?

Reply:
ColorProperty already exists 😑

Reply:
Probably on `material.dart` right? Alright, fair enough!